index.html

```diff
@@ -573,7 +576,7 @@ <h2>Sharing theme</h2>
         timer: null,
       };
 
-      const HINT_DELAY = 15 * 1000;
+      const HINT_DELAY = 1 * 1000; // @@@
```

Comment:
Reminder to remove this before merging (I recognize that is what the @@@ is for)

Reply:
fixed